### PR TITLE
Card integration docs

### PR DESCRIPTION
**Pull request by sarahkoop**
### Summary of changes

 - Document integration steps required for accepting Card payments
 - Following dev doc guidelines markdown template to be transferred easily to official docs site

 ### Checklist

 - ~[ ] Added a changelog entry~

- @sarahkoop

### CODE REVIEW

File: docs/Card/get-started.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (sarahkoop) [inline docs/Card/get-started.md:1]: This header format is required for the dev docs site markdown template

File: docs/Card/get-started.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+sdkVersion: TODO
+---
+
+# Pay with Card
+
+Accept Card payments in your Android app using the PayPal Payments SDK.
+
+## How it works
+
+This diagram shows how your client, your server, and PayPal interact:
```
Comment (sshropshire) [inline docs/Card/get-started.md:16]:
```suggestion
This diagram shows how your client, server, and PayPal interact:
```

File: docs/Card/get-started.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+// TODO - Get a diagram of the payment flow similar to [this](https://developer.paypal.com/braintree/docs/start/overview#how-it-works)
+
+## Eligibility
+
+// TODO - What are the merchant eligibility requirements for Card payments?
+
+This SDK supports a minimum Android API of 23 or higher.
```
Comment (sshropshire) [inline docs/Card/get-started.md]:
```suggestion
This SDK supports a minimum Android API of 21 or higher.
```
😢

File: docs/Card/get-started.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+Android apps can be written in Kotlin or Java 8 or higher.
```
Comment (sshropshire) [inline docs/Card/get-started.md]:
```suggestion
Android apps can be written in Kotlin or Java 8 and higher.
```

File: docs/Card/integration.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+keywords: 
+contentType: docs
+productStatus: current
+apiVersion: TODO
+sdkVersion: TODO
+---
+
+# Pay with Card Custom Integration
+
+Follow these steps to add Card payments.
```
Comment (sshropshire) [inline docs/Card/integration.md]:
```suggestion
Follow these steps to add Card payments:
```

File: docs/Card/integration.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+1. [Know before you code](#know-before-you-code)
+1. [Add Card Payments](#add-card-payments)
+1. [Test and go live](#test-and-go-live)
+
+## Know before you code
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID and generate an access token. 
+
+You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2).
```
Comment (sshropshire) [inline docs/Card/integration.md]:
```suggestion
You will need a server integration to create an order and capture funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). 
```

File: docs/Card/integration.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+        extendedAddress = "Apt. 1A",
+        locality = "city",
+        region = "IL",
+        postalCode = "12345",
+        countryCode = "US"
+    )
+)
+```
+
+Attach the card and the order ID (step 3) to a `CardRequest`.
```
Comment (sshropshire) [inline docs/Card/integration.md]: Might be able to hyperlink to step 3 here?

File: docs/Card/integration.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+
+Approve the order using your `CardClient`.
+
+Call `CardClient#approveOrder` to approve the order, and then handle results:
+
+```kotlin
+cardClient.approveOrder(cardRequest) { result ->
+    when (result) {
+        is CardResult.Success -> {
+            // order was successfully approved with the card and is ready to be captured/authorized (see step 6)
```
Comment (sshropshire) [inline docs/Card/integration.md]:
```suggestion
            // order was successfully approved and is ready to be captured/authorized (see step 6)
```

File: docs/Card/integration.md [hunk 9]
```diff
@@ -0,0 +1,153 @@
...
+        } 
+        is CardResult.Error -> {
+            // handle the error by accessing `result.error`
```
Comment (sshropshire) [inline docs/Card/integration.md]:
```suggestion
            // inspect `result.error` for more information
```

**Review by sshropshire — APPROVED** [review]
LGTM&nbsp;&nbsp;🌊 &nbsp;I left a few nits, take or leave 🙏

**Review by cgdibble — APPROVED** [review]

**Pull request merged**